pages/establishment/details/index.js

```diff
@@ -1,15 +1,44 @@
 const { page } = require('@asl/service/ui');
+const { get } = require('lodash');
+const { form } = require('../../common/routers');
+const schema = require('./schema');
 
 module.exports = settings => {
   const app = page({
     ...settings,
     root: __dirname
   });
 
-  app.get('/', (req, res, next) => {
+  app.use((req, res, next) => {
     req.breadcrumb('establishment.read');
+    req.model = req.establishment;
+    res.locals.static.asruUser = req.user.profile.asruUser;
```

Comment:
This _shouldn't_ be needed anymore if you've switched to `allowedActions`.

Reply:
yes good spot 😅